src/index.js

```diff
@@ -372,5 +376,4 @@ module.exports.main = wrap(main)
   .with(bodyData)
   .with(recordsWrap)
   .with(helixStatus)
-  .with(logger.trace)
```

Comment:
This has to be removed because it will unconditionally catch errors thrown by the Lambda